### PR TITLE
I have added AddAndActivateConnection to NetworkManager

### PR DESCRIPTION
**Pull request by phommel**

### CODE REVIEW

File: NetworkManager.go [hunk 1]
```diff
@@ -91,6 +106,27 @@ func (n *networkManager) ActivateWirelessConnection(c Connection, d Device, ap A
 	return nil
 }
 
+func (n *networkManager) AddAndActivateWirelessConnection(connection map[string]map[string]interface{}, d Device, ap AccessPoint) (ac ActiveConnection, err error) {
+	var opath1 dbus.ObjectPath
+	var opath2 dbus.ObjectPath
+	defer func() {
```
Comment (tom5760) [inline NetworkManager.go]: Just curious, what can panic from this call?  `callError2` returns an error anyway.  Is this necessary?

**Review by tom5760 — CHANGES_REQUESTED** [review]

File: NetworkManager.go (hunk 1 above)
Comment (phommel) [reply NetworkManager.go]: No, it is just some leftover code. Will remove...

Comment (tom5760) [reply NetworkManager.go]: Thanks!

**Review by tom5760 — APPROVED** [review]

**Comment by tom5760** [comment]
Thanks for the contribution!

**Pull request merged**